npm/src/upload/uploadform/display-warning-message.ts

```diff
@@ -0,0 +1,25 @@
+export const rejectUserItemSelection = (
```

Comment:
I'd be tempted to rename the new directory slight to: `npm/src/upload/form/...` 

As the new directory is already nested in the `upload` directory it makes it clear what this form is used for.